containers/clojure/.devcontainer/Dockerfile

```diff
@@ -0,0 +1,76 @@
+# [Choice] Java version (use -bullseye variants on local arm64/Apple Silicon): 11, 16, 11-bullseye, 16-bullseye, 11-buster, 16-buster
```

Comment:
Given Java 17 LTS is out, I assume it makes sense to update?

Reply:
This was on my list! Java 17 is now the default. 😉